src/core/model_config.proto

```diff
@@ -1290,4 +1290,13 @@ message ModelConfig
   //@@     model.
   //@@
   repeated ModelWarmup model_warmup = 16;
+
+  //@@  .. cpp:var:: string custom_library_path
+  //@@
+  //@@     Optional path of the custom library that contains custom
```

Comment:
"Optional paths of the libraries providing custom operations for this model. Valid only for ONNX models."